src/Time/Tags.hpp

```diff
@@ -66,14 +66,24 @@ struct Time : db::SimpleTag {
 
 /// \ingroup DataBoxTagsGroup
 /// \ingroup TimeGroup
-/// \brief Prefix for TimeStepper history
+/// Base tag for the TimeStepper history
+///
+/// \see HistoryEvolvedVariablesTyped
+struct HistoryEvolvedVariables : db::BaseTag {};
```

Comment:
I might be mistaken, but I thought we had agreed to name `BaseTag`s with `BlaBase` and just use `Bla` for the `SimpleTag`s

Reply:
*Obviously* it should be
```c++
template <typename T = void>
struct Tag;

template <>
struct Tag<> : db::BaseTag {};

template <typename T>
struct Tag : Tag<>, db::SimpleTag {
  using type = T;
};
```
(This is one of those ideas that feels dumb but I can't actually put my finger on what is dumb about it.  Sometimes people have actually liked those.)

Reply:
@kidder and I think this suggestion is the right solution (if only C++ would let us write `Tag` instead of `Tag<>`, but alas this is the hand we've been dealt)